package.json

```diff
@@ -38,6 +39,16 @@
       {
         "permissionName": "settings.enabled",
         "displayName": "UI: settings area is enabled"
+      },
+      {
+        "permissionName": "mod-settings.owner.read.stripes-core.prefs.manage",
+        "displayName": "UI: read the user's own preference for the order of links in the main navigation.",
```

Comment:
Could we say something like "UI: read the user's own preferences, such as the order of links in the main navigation"? We are surely going to want to use this for lots of other things, too, once it's in place.